styleguide/components/41-organisms/timeline/_timeline.scss

```diff
@@ -0,0 +1,397 @@
+        top: 0;
+        left: 1.5rem;
+        height: 1rem;
+        border-left: 3px solid;
+        content: '';
+        @include tablet {
+          display: none;
+        }
+      }
+    }
+
+    > dd {
+      @include theme('border-color', 'color-primary', 'timeline-line-color');
+
+      display: flex;
+      position: relative;
+      flex-wrap: wrap;
+      width: 100%;
+      margin: 0 0 0 1.5rem;
+      padding: .5rem calc(#{$gutter-width/2}) 2rem 2.5rem;
+      border-left: 3px solid;
+
+      @include tablet {
+        margin: 0 0 0 11.5rem;
+        padding: 2rem calc(#{$gutter-width/2}) 2rem 5rem;
+      }
+
+      &::before {
+        @include theme('background-color', 'color-secondary', 'timeline-slot-indicator-background');
+        @include theme('border-color', 'color-primary', 'timeline-slot-indicator-border-color');
+
+        display: block;
+        position: absolute;
+        top: .9rem;
+        left: -.5rem;
+        width: .9rem;
+        height: .9rem;
+        border: 3px solid;
+        border-radius: 100%;
+        content: '';
+        z-index: 2;
+
+        @include tablet {
+          top: 2.3rem;
+        }
+      }
+
+      &.active::before {
+        @include theme('background-color', 'color-primary', 'timeline-slot-active-indicator-background');
+      }
+
+      // Video component styling inside a timeline-slot.
+      .video-wrapper {
+        position: relative;
+        width: calc(100% + 4.6rem + 4px);
+        margin-left: calc(-4rem - 4px);
+
+        @include tablet {
+          width: calc(100% + 10rem + 4px);
+          margin-left: calc(-10rem - 4px);
+        }
+
+        // Fix to add whitespace to the video top and bottom areas.
+        &::before,
+        &::after {
+          position: absolute;
+          top: 0;
+          left: 0;
+          width: 100%;
+          height: 100%;
+          background-color: color('white');
+          content: '';
+          z-index: 1;
+        }
+
+        &::after {
+          @include theme('background-color', 'color-secondary', 'timeline-slot-default-background');
+        }
+
+        .video {
+          position: relative;
+          padding-top: 2rem;
+          border: 0;
+          z-index: 2;
+
+          .audio-description {
+            min-height: 2rem;
+            padding-left: 3rem;
+
+            // Dots after a video intersection.
+            &::after {
+              @include theme('border-color', 'color-primary', 'timeline-slot-active-indicator-background');
+
+              position: absolute;
+              bottom: .2rem;
+              left: calc(1.5rem + 1px);
+              height: 1rem;
+              border-left: 3px dotted;
+              content: '';
+              z-index: 10;
+
+              @include tablet {
+                left: calc(5rem + 1px);
+              }
+            }
+          }
+        }
+
+        .responsive-video {
+          // Dots before a video intersection.
+          &::before {
```

Comment:
Combine this in one query with the other dot.
Better for maintenance.

Reply:
👍